lib/ex_doc/config.ex

```diff
@@ -42,7 +42,8 @@ defmodule ExDoc.Config do
             source_url: nil,
             source_url_pattern: nil,
             title: nil,
-            version: nil
+            version: nil,
+            warn_on_undefined_functions: true
```

Comment:
hm, it's actually gonna warn on undefined callbacks and types. Maybe it should be simply `warn_on_undefined`? 

Btw, I only added the option to be able to disable it in integration tests.